onvm/go.sh

```diff
@@ -72,11 +98,112 @@ while getopts "a:r:d:s:t:l:p:z:cv" opt; do
         z) stats_sleep_time="-z $OPTARG";;
         c) shared_cpu_flag="-c";;
         v) verbosity=$((verbosity+1));;
+        m)
+            if [[ -n $cpu ]]
+            then
+                echo "Error: Cannot use manual manager core flag with legacy syntax. Correct syntax: "
+                echo "  ./go.sh -k <port mask> -n <NF cores> [OPTIONS]"
+                exit 1
+            else
+                cpu=$OPTARG
+            fi;;
+        k)
+            if [[ -n $ports ]]
+            then
+                echo "Error: Cannot use manual port mask flag with legacy syntax. Correct syntax: "
+                echo "  ./go.sh -k <port mask> -n <NF cores> [OPTIONS]"
+                exit 1
+            else
+                ports=$OPTARG
+            fi;; 
+        n) 
+            if [[ -n $nf_cores ]]
+            then
+                echo "Error: Cannot use manual NF core mask flag with legacy syntax. Correct syntax: "
+                echo "  ./go.sh -k <port mask> -n <NF cores> [OPTIONS]"
+                exit 1
+            else
+                nf_cores=$OPTARG
+            fi;;
         \?) echo "Unknown option -$OPTARG" && usage
             ;;
     esac
 done
 
+# Verify that dependency bc is installed
+if [[ $(dpkg-query -W -f='${Status}' bc 2>/dev/null | grep -c "ok installed") == 0 ]]
```

Comment:
mentioned this in #229 , but it probably should be fixed here

Reply:
Fixed in most recent commit.